setup.py

```diff
@@ -3,10 +3,10 @@
 import os
 
 # Put here required packages
-packages = ['Django==1.7',
+packages = ['Django==1.8',
     'static3', # serve static files in the server
-    'djangorestframework',
-    'django-filter',
+    'djangorestframework==2.4.8',
```

Comment:
Do we want to be slightly more flexible on versions as suggested in [python's packaging guidelines](https://packaging.python.org/installing/#installing-from-pypi)?